src/balena-model.ts

```diff
@@ -1076,10 +1071,6 @@ export interface Release {
 		known_issue_list: Types['Text']['Write'] | null;
 		note: Types['Text']['Write'] | null;
 		invalidation_reason: Types['Text']['Write'] | null;
-		is_final: Types['Boolean']['Write'];
-		semver: Types['Short Text']['Write'];
```

Comment:
Just noting that these two are actually write-able on the OData API level (handled in hooks), but of course the model can't know that and we would need to augment them manually if needed.